apps/hub-app/src/main.tsx

```diff
@@ -0,0 +1,14 @@
+import { StrictMode } from 'react';
+import * as ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
+
+import App from './app/App';
+
+ReactDOM.render(
+  <StrictMode>
+    <BrowserRouter>
```

Comment:
We should probably do some tests this week with skynet deployments and routing. We might need to switch to HashRouter to enable deep linking into routes. This was the case with other ipfs deployments in the past.

Reply:
Yes, we will need to use hash router. Skynet doesn't support browser router